Food/jcli.lua

```diff
@@ -0,0 +1,50 @@
+local name = "jcli"
+local version = "0.0.16"
+
+food = {
+    name = name,
+    description = "Jenkins CLI allows you manage your Jenkins as an easy way",
+    license = "MIT",
+    homepage = "https://jenkins-zh.cn",
+    version = version,
+    packages = {
+        {
+            os = "darwin",
+            arch = "amd64",
+            url = "https://github.com/linuxsuren/jenkins-cli/releases/download/v" .. version .. "/" .. name .. "-darwin-amd64.tar.gz",
```

Comment:
If you change this to

```
url = "https://github.com/linuxsuren/jenkins-cli/releases/download/v" .. version .. "/" .. name .. "-" .. os .. "-" .. arch .. ".tar.gz",
```

Then it should work. Your issue in c6bdb826337 was the missing hyphen between `os` and `arch` - you just separated it with an empty string, causing the result to be `jcli-darwinamd64` instead of `jcli-darwin-amd64`.

The same can be applied to lines 27 and 40.

Reply:
Thanks.

Reply:
The error still exists.